koi_api/resources/instance.py

```diff
@@ -227,7 +233,10 @@ def head(
         me,
     ):
         return SUCCESS(
-            "", last_modified=instance.instance_last_modified, valid_seconds=LT_INSTANCE
+            "",
+            last_modified=instance.instance_last_modified,
+            valid_seconds=LT_INSTANCE,
+            etag=instance.instance_etag,

```

Comment:
we should also return the eTag in the GET response. Likewise for all the other models